lib/connect/api/assetRequests/create.js

```diff
@@ -111,13 +111,31 @@ const createRequestForProduct = async (client, inputData, productId, assetExtera
 };
 
 const createAssetRequestFromOrder = async (client, data) => {
+  if (!data.customer_external_uid) {
+    const extUid = await lookupTierByExternalId(client, data.customer_external_id, data.hub_id);
+    if (extUid) {
```

Comment:
This if is not needed.

data. customer_external_uid = await lookupTierByExternalId(client, data.customer_external_id, data.hub_id);

The lookup returns "null" if it can't find the tier account.